python/ray/includes/common.pxi

```diff
@@ -0,0 +1,70 @@
+from cpython cimport Py_buffer, PyBytes_FromStringAndSize
+from libc.stdint cimport int64_t, uintptr_t
+from libcpp.memory cimport shared_ptr
+
+from ray.includes.common cimport CBuffer
+
+
+cdef class Buffer:
```

Comment:
This is worth documenting the purpose so that people know this exists to pass `RayObject`s into pyarrow by wrapping them in a Python object that implements the buffer interface.